tools/runners/run-mongo.js

```diff
@@ -537,6 +536,11 @@ var launchMongo = function (options) {
         maybeReadyToTalk();
       }
 
+      if (/[ReplicationExecutor] Locally stored replica set configuration does not have a valid entry for the current node/) {
```

Comment:
Hmm... the `[` and `]` aren't escaped so it would never match the actual line but the regex itself doesn't test _against_ anything (à la `.test(data)`) so it would be setting `replSetReadyToBeInitiated` on every log line.  Maybe it's actually fine to just always reconfig?  Heh.